internal/services/elastic/elastic.go

```diff
@@ -197,158 +193,3 @@ func (s *Service) GetTotalDistanceDriven(ctx context.Context, deviceID string) (
 
 	return body, nil
 }
-
```

Comment:
In this file, you can delete these as well
```go
type GetHistoryParams struct {...
	
func (g *GetHistoryParams) SetDefaultHistoryParams() {
```

Reply:
Ah, thanks, careless of me.